OracleDatabase/RAC/OracleRealApplicationClusters/dockerfiles/12.2.0.1/sshUserSetup.sh

```diff
@@ -16,7 +16,7 @@
 # confirmations.
 #Please note that the script would remove write permissions on the remote hosts
 #for the user home directory and ~/.ssh directory for "group" and "others". This
-# is an SSH requirement. The user would be explicitly informed about this by teh script and prompted to continue. In case the user presses no, the script would exit. In case the user does not want to be prompted, he can use -confirm option.
+# is an SSH requirement. The user would be explicitly informed about this by teh script and prompted to continue. In case the user presses no, the script would exit. In case the user does not want to be prompted, they can use the -confirm option.
```

Comment:
teh -> the

Reply:
We should probably run the whole file past one of our doc. writers.  "In case the user does not want to be prompted, they can use the -confirm option." might be better worded as "Use the -confirm option to skip confirmation prompts."  Then we also don't have the singular/plural confusion to work around being sensitive to gender.

Reply:
And the `-confirm` option should be formatted as monotype using back ticks.